src/index.js

```diff
@@ -1,49 +1,50 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
+import React from "react";
+import ReactDOM from "react-dom";
 
-import ToDo from './todo';
-import reportWebVitals from './reportWebVitals';
-import {createStore} from 'redux';
-import {Provider} from 'react-redux';
+import ToDo from "./todo";
+import reportWebVitals from "./reportWebVitals";
+import { createStore } from "redux";
+import { connect, Provider } from "react-redux";
 
-
-
-const counter=(state=0, action)=>{
-  switch(action.type){
-    case 'INCREMENT':
+const counter = (state = 0, action) => {
+  switch (action.type) {
+    case "INCREMENT":
       return state + 1;
-    case 'DECREMENT':
+    case "DECREMENT":
       return state - 1;
     default:
       return state;
   }
-} 
-const store= createStore(counter);
-const ToDoMain=()=>(
+};
+const store = createStore(counter);
+
+const ToDoMain = (props) => (
   <div>
     <ToDo
-      value={store.getState()}
```

Comment:
Aneke dile, component rerender nohoi. Karon ai component to redux'r logot bind huwa nai.